lnd_test.go

```diff
@@ -7919,7 +7946,7 @@ var testsCases = []*testCase{
 		name: "update channel policy",
 		test: testUpdateChannelPolicy,
 	},
-	{
+	/*{
```

Comment:
not intentional?

Reply:
Heh yeh, had isolated that test in order to debug it.